mailbox/store/src/test/java/org/apache/james/mailbox/store/StoreMessageIdManagerTestSystem.java

```diff
@@ -59,9 +60,10 @@ public StoreMessageIdManagerTestSystem(MessageIdManager messageIdManager, Mailbo
     @Override
     public MessageId persist(MailboxId mailboxId, Flags flags) {
         MessageId messageId = messageIdFactory.generate();
+        MessageUid uid = MessageUid.of(new Random().nextLong());
```

Comment:
We try to not put any random value in our test, to make them reproducible.
+ here you risk a collision with 2 identical messageUid